_stories/molecules/Breadcrumbs/README.md

```diff
@@ -2,11 +2,7 @@
 
 ### **NOTE**: The component will only render with a minimum width of **991px**
 
-**If the default <Breadcrumbs> component does not suit your needs, see `<BreadcrumbsWrapper/>` an unopinionated wrapper that lets you pass any random content you want, so you can create an array with the exact breadcrumbs to display.**
-
-The `<Breadcrumbs>` component creates navigation links for the current pathname based on an opinionated configuration object.
-The idea behind this component is to drop it anywhere with minimal or no configuration and get breadcrumbs generated automatically.
-The only required prop is a `pathname`, but a `config` object is also accepted for fine tunning.
+The <Breadcrumbs> component creates navigation links for the current pathname based on an opinionated configuration object. The idea behind this component is to drop it anywhere with minimal or no configuration and get breadcrumbs generated automatically. The only required prop is a pathname, but a config object is also accepted for fine tuning. The library also has a <BreadcrumbsWrapper> component if you need to customize the implementation yourself.
```

Comment:
The backticks got removed from `<Breadcrumbs>` and `<BreadcrumbsWrapper>`. Need those so the md is rendered properly. Also nice to have on `pathname` and `config`.

Reply:
Fixed